docs/source/quickstart.ipynb

```diff
@@ -127,7 +127,9 @@
    "id": "fbca1394",
    "metadata": {},
    "source": [
-    "RP uses **MongoDB** as one of the communication layers to store and retrieve operational data during the execution of an application. The MongoDB server must be reachable from **both**, the host that runs the RP application and the target resource which runs the pilots. \n",
+    "### x.2.1. `MongoDB`\n",
```

Comment:
update style, since "MongoDB" is not a command here, but a service name -> remove backquotes